speaking-portal-project/src/animationFactory/animationProcessor.ts

```diff
@@ -29,23 +31,25 @@ export async function mouthCuesToInputFile({
 /*
 This converts  data from the MouthCue type into a format for ffmpeg
 */
-export function generateFrameData(avatar: string, mouthCues: MouthCue[]) {
+export function generateFrameData(avatar: string, mouthCues: MouthCue[], timer: Timer) {
     let character = new Avatar(avatar)
     let frameData = ''
 
-    console.log('Adding realism...')
+    console.log(`${timer.getTotalTimeElapsed()} - Adding realism...`)
```

Comment:
So I think there are Logger libraries that could allow us to make up console.timer()...

Which could trigger when a process.Env variable is set to 'true' or something. This would make it so that we could define different logging under different circumstances, such as if we are 'testing' performance.

But we would have to look at this more, not totally sure if that would be route to go

Reply:
I wasn't sure if we wanted to add another library if we were doing basic checks to see if we could make specific processes more efficient. Now that we have quantitative data on which processes are quick, I agree we can definitely remove those timers. I'll make it an issue for next week so that we get the stats for this week at least. It's nice that we know where to focus our attention now!

Reply:
Also, we should include times for phonetics if we haven't already. When I was testing it with the server the phonetic recognizer cleaned house compared to the English one.

Reply:
We can make this an issue for next week to actually log our results! This way we can let the client know the actual difference between how quickly we can get it to run with lower lip-sync quality vs. the higher lip-sync quality which takes more time. Then we can ask him to make the decision.